src/components/toolbar/tile-toolbar-button.tsx

```diff
@@ -12,16 +25,22 @@ export interface TileToolbarButtonProps {
  * A generic, simple button that can go on a tile toolbar.
  */
 export const TileToolbarButton =
-  function({name, onClick, selected, disabled, children}: PropsWithChildren<TileToolbarButtonProps>) {
+  function({name, title, keyHint, onClick, selected, disabled, children}: PropsWithChildren<TileToolbarButtonProps>) {
+
+    const tipOptions = useTooltipOptions();
 
+    const tooltip = formatTooltip(title, keyHint);
     return (
+      <Tooltip key={name} title={tooltip} {...tipOptions} >
```

Comment:
The key might not be necessary here anymore since this TileToolbarButton only has one Tooltip child. If there were a collection of generic TileToolbarButton components then each of those would need a key. However it seems we almost always wrap the TileToolbarButton in a parent component that seems to have a 1:1 correspondence with the `name`.  Anyhow it won't hurt to leave the key here since we've got the name anyhow.

Reply:
Removed.